app.js

```diff
@@ -21,24 +21,39 @@ oppiaGithubPages.config(['$routeProvider', function($routeProvider) {
       controller: 'mainController',
       projectTag: 'A tool for creating interactive tutors'
     })
-    .when('/wiki/embedded', {
-      templateUrl: 'pages/embedded.html',
+    .when('/wiki/Embedded', {
```

Comment:
Do we ever use the non-wiki namespace (i.e. do we have any pointers to /*/* where the first '*' is something other than 'wiki')? If not, maybe we don't need the '/wiki' part.